opentracing-spring-cloud/pom.xml

```diff
@@ -51,6 +51,26 @@
     </dependency>
 
     <dependency>
+      <groupId>org.springframework</groupId>
+      <artifactId>spring-jms</artifactId>
```

Comment:
Rabbit/Ampq stuff will need to be supported differently.
I guess we'll need a new instrumentation project -- if we don't have now.
The project will need to be similar to what Sleuth does with Spring's msging abstraction.